auth-internal-api/build.gradle

```diff
@@ -0,0 +1,16 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    maven { url = "$repoUrl" }
+    jcenter()
+  }
+  dependencies {
+    classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
+  }
+}
+
+dependencies {
+  compile project(':common')
+  testCompile group: 'junit', name: 'junit', version: junitVersion
```

Comment:
I think the junit impl is defined in the parent gradle file. Please try to rely on that.